test/resources/layout.css

```diff
@@ -174,3 +174,28 @@ body {
   border: none;
   font-size: 14px;
 }
+
+.test-sidenav-list {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  border: 0;
+  list-style: none;
+  padding-left: 1rem;
+}
+
+.test-sidenav-list__item{
+  box-sizing: border-box;
+  font-size: 14px;
+  padding: 0;
+  margin: 0;
+  border: 0;
+  list-style: none;
+}
```

Comment:
These shouldn't be necessary, should they?  The playground is supposed to demonstrate the component and its styles.  The "test" classes are for extraneous components included within another component (_e.g._ buttons in an action bar or modal, icons in a panel, etc.)

Reply:
The Side Nav component is:
- setting the width of the component in expanded state and in condensed state
- adding 2 sections - Main and Utility 

What is inside the 2 sections is the Nested List which is "external" to Side Nav and that's why I use 
`<ul class="test-sidenav-list">
        <li class="test-sidenav-list__item">Item 1</li>
        <li class="test-sidenav-list__item">Item 1</li>
        <li class="test-sidenav-list__item">Item 1</li>
        <li class="test-sidenav-list__item">Item 1</li>
 </ul>`
just because I need "content" to test the Side Nav.

I am struggling to understand your comment above ("Appears that a few things aren't self-contained for styling yet (see screenshot).")